templates/index.html

```diff
@@ -54,14 +54,12 @@ <h3>VMs for Windows, macOS and Linux</h3>
 <section class="p-strip--suru-accent">
   <div class="row u-equal-height u-vertically-center">
     <div class="col-6 u-align--center u-hide--small">
-      <a aria-controls="modal" href="https://assets.ubuntu.com/v1/bb21c0e8-Ubuntu+Server+CLI+pro+tips+06.01.20.pdf?_ga=2.194029293.1707463146.1680603251-187335696.1680603251" aria-hidden="true" tabindex="-1">
```

Comment:
I think you are supposed to keep that `_ga...` parameter for tracking. You can confirm with @laszlokajtar

Reply:
Why would we be tracking it here though? Clicking on the image doesn't do anything nor does it route anywhere